src/actions/removeFeature.js

```diff
@@ -0,0 +1,5 @@
+export const REMOVE_FEATURE = "REMOVE_FEATURE";
+
+export const removeFeature = (target) => {
+    return { type: REMOVE_FEATURE, feature: target };
```

Comment:
If having feature there makes sense to you keep it! I would just add a comment around it letting other devs know that it's what they'll typically see as the 'payload'